uk.co.compucorp.civicrm.hremails/email-templates/src/assets/scss/app.scss

```diff
@@ -1,4 +1,7 @@
+@import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/overrides/variables";
```

Comment:
no, in bootstrap sass the overrides come first, because of the usage of the `!default` property (https://robots.thoughtbot.com/sass-default)